File: Working Hour or Not/Working Hour or Not.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Oct  5 11:40:37 2020

@author: ravi
"""

start,end = map(int,input("Please Enter Working Hour in 24 Hour Format\nStart time and End Time ==> ").split())

if start>=9 and end<=18:
    print("Working Hour")
else:
    print("Not a Working Hour")

File: Working Hour or Not/Working Hour or Not.cpp
#include<iostream>
using namespace std;

int main(){
    int start,end;
    cout<<"Please enter Working hour in 24 Hour format.\nStart time and End time ==> ";
    cin>>start>>end;

    if(start >= 9 && end<=18){
        cout<<"Working Hour";
    }else{
        cout<<"Not a Working Hour";
    }
    return 0;   
}